components/prism-rescript.js

```diff
@@ -0,0 +1,55 @@
+Prism.languages.rescript = {
+	'comment': {
+		pattern: /\/\/.*|\/\*[\s\S]*?(?:\*\/|$)/,
+		greedy: true
+	},
+	'char': { pattern: /'.'/, greedy: true },
+	'string': {
+		pattern: /"(?:\\(?:\r\n|[\s\S])|[^\\\r\n"])*"/,
+		greedy: true
+	},
+	'class-name': /\b[A-Z]\w*|@[a-z.]*|#[A-Za-z]\w*|#\d/,
+	'function': /[a-zA-Z]\w*(?=\()|(\.)[a-z]\w*/,
```

Comment:
CI found another unused capturing group. It's probably meant to be a lookbehind, right?
```suggestion
	'function': { pattern: /[a-zA-Z]\w*(?=\()|(\.)[a-z]\w*/, lookbehind: true },
```

Reply:
Right

Reply:
Done